hardware/camera/Android.mk

```diff
@@ -1,21 +1,31 @@
+# Create symlinks for camera libraries. Currently only applicable
+# to sdm845 boards.
+
+# Warning: The same guard is used in common.mk, where this module
+# is depended on.
 ifneq ($(filter sdm845,$(TARGET_BOARD_PLATFORM)),)
```

Comment:
why not just drop this guard?

Reply:
These guards had to be dropped from `common.mk` instead, because these variables (`TARGET_BUILD_PLATFORM` and `WIFI_DRIVER_BUILT`) are set *after* `common.mk` has been fully included.
Luckily, adding modules to `PRODUCT_PACKAGES` does not result in errors when these do not exist (which is weird considering this can easily lead to typos and other accidents).

Reply:
I think the build system only complains about missing modules if `LOCAL_MODULE_TAGS := optional` is not set?